rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -905,7 +938,9 @@ def test_latest_upgrade_adds_remove_tables(user_data_dir):
     views_before = {x[0] for x in result}
 
     last_db.logout()
+
     # Execute upgrade
+    _use_prepared_db(user_data_dir, base_database)
```

Comment:
Again ... why are you adding this? If it's for the same reason as before fix it with a patch which will be temporary for this version only.